README.md

```diff
@@ -1,52 +1,50 @@
+## What is Novus
+NovusCore is an MMO Engine.
+
+Currently Novus is developing a game/server solution where our main focus is to be feature complete with the most popular MMOs out there.
+Long term, we hope to support many more features.
+
+## The Novus Promise
+The project was made with the promise that we would always focus on reliability, redundancy, and performance over convenience. We achieve this through making use of experience, but also applying modern techniques and design patterns.
+
+The end-goal is to provide a game/server setup, capable of tackling all of the limitations set by the current standard. Solving those issues are complicated, but we start by applying a proper foundation for our architecture to allow for better flow of information(data) and performance.
+
+## Novus Discord
+The project has an official [Discord](https://discord.gg/gz6FMZa).
+You will find the developers to be active on the discord and always up for answering any questions you might have regarding the project. Despite Novus not currently being ready for production level use, we are always welcoming any users that want to try using it.
+
+## Dependencies
+* [OpenSSL 1.1.0](https://www.openssl.org/source/)
+* [Premake5](https://premake.github.io/)
+* [Vulkan 1.1 (or higher)](https://vulkan.lunarg.com/)
+
+## Libraries
+Here we include a honorable mention to all the libraries included directly into the source of Novus. You do not need to download these on your own, but we felt it was important to address these as without them, Novus would be a lot more time consuming to develop.
+- [All Libraries from Engine](https://github.com/novusengine/Engine)
+- [Jolt](https://github.com/jrouwe/JoltPhysics)
+- [Casc](https://github.com/heksesang/CascLib)
+- [Cuttlefish](https://github.com/akb825/Cuttlefish)
+
 ## How to build
-1. Fork and clone the repositry
-2. Open the project folder and open a terminal within the project's directory.
-3. Use the command `premake5 vs2022`
-   - You can change the files generated by using a different [action](https://premake.github.io/docs/using-premake) than `vs2022`.   
+>[!TIP]
+> It's recommended to fork, clone, and build [Engine](https://github.com/novusengine/Engine) first and use a [symlink](https://schinagl.priv.at/nt/hardlinkshellext/linkshellextension.html) to link the engine into the `Submodule` folder of the Asset Converter.
```

Comment:
We actually do not recommend the workflow of symlinking a locally cloned Engine repo in the Submodule folder of game.

It is convenient for development, but you lose the ability to update the submodule and push that to the Game/Assetconverter repo. Now you have to symlink it regularly after updating the submodule. It probably works, but I think this could lead to issues later on so I don't think it should be recommended.